client/public/js/main.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function(){
```

Comment:
Why are not using reactjs for this?

Reply:
Didn't originally think to use it, I would see how to implement with reactjs